src/clj_momo/lib/es/index.clj

```diff
@@ -19,6 +19,29 @@
   "make a template uri from a host and a template name"
   (format "%s/_template/%s" uri template-name))
 
+(s/defn rollover-uri :- s/Str
+  "make a rollover uri from a host and an index name"
+  ([uri alias] (rollover-uri uri alias nil false))
+  ([uri :- s/Str
+    alias :- s/Str
+    new-index-name :- (s/maybe s/Str)
+    dry_run :- s/Bool]
+   (str (index-uri uri alias)
+        "/_rollover"
+        (when new-index-name
+          (str "/" new-index-name))
+        (when dry_run
+          "?dry_run"))))
+
+(s/defn refresh-uri :- s/Str
+  "make a refresh uri from a host, and optionally an index name"
+  [uri :- s/Str
+   index-name :- (s/maybe s/Str)]
+  (str uri
+       (when index-name
```

Comment:
same `cond->` remark

Reply:
There is only 1 `when` here and we need to add `"/_refresh"` using `cond->` would make be something like this:
```clojure 
 (cond-> uri
       index-name (str "/" index-name)
       true               "/_refresh")
```
if you think it's clearer I can change it.